ucsb-cs156-f24/proj-happycows-f24-09
# NM - made feature make announcements form and table visible on play page

## Overview

In this PR, I created pagedAnnouncementFixtures.js, PagedAnnouncementTable.js, and PagedAnnouncementTable.test.js  to show the announcements

## Screenshots (Optional)

<img width="1077" alt="Screenshot 2024-11-26 at 4 31 52 PM" src="https://github.com/user-attachments/assets/b2f4af52-0a76-4794-8caa-8340569f0073">


## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go to the play page and check out the announcements. toggle with the show and hide.
4.  You can also check the dokku deployment: https://happycows-dev-sexyjesusfreak.dokku-09.cs.ucsb.edu

## Tests


## Linked Issues

Closes #39
